requirements.txt

```diff
@@ -8,4 +8,7 @@ openmeteo-requests==1.2.0
 requests-cache==1.2.0
 retry-requests==2.0.0
 gdown==5.1.0
-xgboost==2.0.3
\ No newline at end of file
+xgboost==2.0.3
+plotly
```

Comment:
Sorry small thing but can you remove `plotly` and`streamlit` for now? They were just needed for the frontend bit right so can add them in later?

Reply:
> Sorry small thing but can you remove `plotly` and`streamlit` for now? They were just needed for the frontend bit right so can add them in later?

Plotly is needed for the graph even in the CLI version, so should i just remove streamlit for now?

Reply:
> Sorry small thing but can you remove `plotly` and`streamlit` for now? They were just needed for the frontend bit right so can add them in later?

removed streamlit